### ISSUE TITLE
Compile against Scala 2.13.0

### ISSUE THREAD

**Issue by sidnt** [bot4s/telegram#78]

**Comment by mukel** [comment]
We are blocked by some dependencies, as soon as the dependencies are published for 2.13 we are good to go.

**Comment by mukel** [comment]
Upgrade on hold till https://github.com/jokade/slogging/issues/39 is updated.

**Comment by dennypenta** [comment]
Is it a reason why I'm getting an error on compile?
`sbt.librarymanagement.ResolveException: Error downloading com.bot4s:telegram-core_2.13:4.4.0-RC1`

**Comment by mukel** [comment]
There's no 2.13 release yet, there's only [slogging](https://github.com/jokade/slogging) holding us down; it seems is abandoned. Maybe it's time to move to [Log4s](https://github.com/Log4s/log4s)?.
The logging library must support 2.11, 2.12, 2.13 and ScalaJS. **Any alternative suggestions?**

**Comment by morozov11** [comment]
Maybe [com.typesafe.scala-logging](https://github.com/lightbend/scala-logging) ?

**Comment by holinov** [comment]
maybe logstage?

**Comment by zakolenko** [comment]
What do you think about abstracting over the logger, so that the user can choose the appropriate logger implementation on their own?

**Comment by holinov** [comment]
https://github.com/bot4s/telegram/pull/88
i'v started work on `2.13`
it would be great if any one helps

**Comment by eugeniyk** [comment]
`2.13.2` is out, let's move ;)

**Comment by ex0ns** [comment]
Any news regarding this upgrade ?
I can take care of the migration if we decide which logger we should be using. I like lightbend/scala-logging a lot but I never tested it for Scala.js (does it even work ?)

**Comment by mukel** [comment]
Scala feels right now like the Bates computer scene from "The Onion Movie". I started updating to 2.13 but then had to drop 2.11 and also ScalaJS support, rotting dependencies, it's maintenance hell and Dotty is just around the corner. Updating the project to 2.13 is a very welcome contribution; support for 2.12 is a hard requirement, you can drop ScalaJS and 2.11. Switching to lightbend/scala-logging is also OK for me.

**Comment by ex0ns** [comment]
First step to a stable 2.13 release was done in #94. Closing this issue
